<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="pageApp">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>367</width>
        <height>160</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Application settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxCanClose">
             <property name="text">
              <string>To tray if closed</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDatabase">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>367</width>
        <height>160</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Database</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBoxDatabases"/>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxMySQL">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>MySQL configuration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditMySQLUsername"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelMySQLUsername">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelMySQLHostname">
            <property name="text">
             <string>Hostname</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditMySQLHostname"/>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEditMySQLPassword">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEditMySQLDatabase"/>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="labelMySQLPassword">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelMySQLDatabase">
            <property name="text">
             <string>Database:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageSMTP">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>367</width>
        <height>160</height>
       </rect>
      </property>
      <attribute name="label">
       <string>SMTP Warning configuration</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSMTP">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="0">
           <widget class="QLabel" name="labelSMTPHostname">
            <property name="text">
             <string>Hostname</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEditSMTPPassword">
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="labelSMTPPassword">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelSMTPUsername">
            <property name="text">
             <string>Username:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLineEdit" name="lineEditSMTPUsername"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditSMTPHostname"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageAlarmWarming">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>367</width>
        <height>160</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Alarm warming</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBoxSounds"/>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButtonOpenFile">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Open file</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPostponeInterval">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Postpone interval:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QTimeEdit" name="timeEditPostpone">
         <property name="displayFormat">
          <string>hh:mm:ss</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelVolume">
         <property name="text">
          <string>Alarm volume:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QSlider" name="sliderVolume">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageInterfaceColors">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>367</width>
        <height>160</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Interface colors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonSetExpiredScheduleColor">
         <property name="text">
          <string>Set expired schedules color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelExpiredScheduleColor">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkBoxCanClose</tabstop>
  <tabstop>comboBoxDatabases</tabstop>
  <tabstop>lineEditMySQLHostname</tabstop>
  <tabstop>lineEditMySQLDatabase</tabstop>
  <tabstop>lineEditMySQLUsername</tabstop>
  <tabstop>lineEditMySQLPassword</tabstop>
  <tabstop>lineEditSMTPHostname</tabstop>
  <tabstop>lineEditSMTPUsername</tabstop>
  <tabstop>lineEditSMTPPassword</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
